platform/node/DEVELOPING.md

```diff
@@ -8,25 +8,31 @@ To develop these bindings, you’ll need to build them from source. Building req
 the [macOS](../macos/INSTALL.md#requirements) or [Linux](../linux/README.md#prerequisites) install documentation, depending
 on the target platform.
 
-To compile the Node.js bindings and install module dependencies, from the repository root directory, run:
+To compile the Node.js bindings and install module dependencies, from the repository root directory, first run:
 
-    npm install --build-from-source
+```
+make distclean
+```
+
+If you are rebuilding after time has passed.
 
-To recompile just the C++ code while developing, run `make node`.
+Then do:
 
-To create an Xcode project and use a GUI debugger in the case of a crash, run `make xnode`.
+```bash
+make node
```

Comment:
It appears that `make xnode` does not exist anymore so I removed reference to it. Not sure when the `xnode` target was removed /cc @friedbunny @tmpsantos